src/Tribe/Template.php

```diff
@@ -263,10 +263,10 @@ public function set_template_context_extract( $value = false ) {
 	 *
 	 * @return array
 	 */
-	public function merge_context( $context = array(), $file = null, $name = null ) {
+	public function merge_context($context = [], $file = null, $name = null ) {
 		// Allow for simple null usage as well as array() for nothing
 		if ( is_null( $context ) ) {
-			$context = array();
+			$context = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: c44c5f8dda90f0e7cee6462f1082b9d8</sub>